macros/csssyntax.ejs

```diff
@@ -386,9 +386,11 @@ function formatTypesSyntax(formattedSyntax, describedTypes) {
             types.forEach(function(type, index, types) {
                 // Avoid recursions by checking whether a type was already
                 // described before
+                // check whether https://github.com/mdn/data/pull/66 was merged
+                var syntax = data.syntaxes[type].syntax || data.syntaxes[type];
```

Comment:
👍 works with current data, and appears that it will work with https://github.com/mdn/data/pull/66.